config/crd/snapshot.storage.k8s.io_volumesnapshotclasses.yaml

```diff
@@ -4,19 +4,31 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: (devel)
-    api-approved.kubernetes.io: "https://github.com/kubernetes-csi/external-snapshotter/pull/139"
+    controller-gen.kubebuilder.io/version: v0.2.4
```

Comment:
We need to add this:
api-approved.kubernetes.io: <your PR 256>

Reply:
I've included this line.